index.md

```diff
@@ -3,15 +3,55 @@ layout: lesson
 root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
-FIXME: home page introduction
 
-<!-- this is an html comment -->
+This tutorial is built to teach you ESMValTool.  
```

Comment:
I would add a little more explanation about the tutorial and ESMValTool. One or two sentences what ESMValTool is so that people that stumble on this page via a shared link or search engine can also find out what it is about. 
I think it is also good to mention that this tutorial is meant to teach the basics of ESMValTool, or if we also add advanced sections, to tell people about this distinction and how to use this tutorial. (e.g. "Section 0 trough 7 are meant to teach the basics and get you started on using ESMValTool, these sections are meant to be completed one after another as a complete tutorial. For more advanced topics like writing your own diagnostic scripts or contributing to the community, we refer you to the advanced sections, which can be done in any order.)